integrations/gcp/gcp_core/gcp_client.py

```diff
@@ -0,0 +1,107 @@
+from typing import Any, AsyncGenerator, Optional
+
+from google.api_core.exceptions import PermissionDenied
+from google.cloud.asset_v1 import (
+    AssetServiceAsyncClient,
+    ContentType,
+    ListFeedsRequest,
+)
+from google.cloud.asset_v1.services.asset_service import pagers
+from loguru import logger
+from port_ocean.context.event import event
+from port_ocean.context.ocean import ocean
+
+from gcp_core.types import CloudAssetInventoryFeed
+from gcp_core.utils import parseProtobufMessages
+
+RESOURCE_METADATA_CONTENT_TYPE = ContentType(1)
```

Comment:
what does it mean? would user will want to change it?

Reply:
This refers to what we want the feed to include in every event. Since it doesn't the user in any way I put it here. The user should have no reason to change this.